packages/core/src/telemetry/service.ts

```diff
@@ -69,6 +68,16 @@ export class TelemetryService {
     this.queue.add(() => this.processEvent());
   }
 
+  private establishHeartbeat() {
+    this.heartbeatIntervalId = setInterval(() => {
+      this.record({ event: "Heartbeat" });
+    }, 60_000);
+  }
+
+  private killHeartbeat() {
+    clearInterval(this.heartbeatIntervalId);
+  }
```

Comment:
Style nit: I'd probably just inline these (create the interval in the constructor, clear it in `kill()`)